.travis.yml

```diff
@@ -19,6 +19,17 @@ matrix:
     os: windows
     rust: nightly
     script: cargo build --workspace --verbose --release
+  - name: Centos Binary
+    services: docker
+    env:
+      - OS_TYPE=centos OS_VERSION=7
+    before_install:
+      - sudo apt-get update
+      - echo 'DOCKER_OPTS="-H tcp://127.0.0.1:2375 -H unix:///var/run/docker.sock -s devicemapper"' | sudo tee /etc/default/docker > /dev/null
+      - sudo service docker restart
+      - sudo docker pull centos:centos${OS_VERSION}
+    rust: nightly
+    script: cargo build --workspace --verbose --release
```

Comment:
all this does is run the cargo build command after pulling a docker image. need to run the command IN the docker image i think. 

also does this successfully publish a release for centos7 when tagged? Building for it is a good sign but ideally we need to upload the binary as well. Take a look at the `deploy` and `before_deploy` keys. We need to get the packaged binary from the docker build and zip that in a separate release for centos7.

Reply:
Could you just build and release the binary built on CentOS 7 instead of having two types of Linux binary? from a little testing I did, the CentOS built on will run on Ubuntu.

Reply:
ack, we can do this for future releases just to consolidate